daemon/src/main.rs

```diff
@@ -43,12 +44,9 @@ enum Commands {
     Daemon {
         /// The directory to sync. Defaults to current directory.
         directory: Option<PathBuf>,
-        /// Multiaddr of a peer to connect to.
+        /// Read <node_id>#<passphrase> from stdin.
```

Comment:
This is user visible documentation, isn't it? So I'd rephrase it to be a bit less technical:
```suggestion
        /// Connect to another peer. Will ask for <node_id>#<passphrase> to connect to.
```